crates/analytics/src/auth_events/metrics/authentication_attempt_count.rs

```diff
@@ -29,19 +27,13 @@ where
 {
     async fn load_metrics(
         &self,
-        dimensions: &[SdkEventDimensions],
+        _merchant_id: &str,
         publishable_key: &str,
-        filters: &SdkEventFilters,
         granularity: &Option<Granularity>,
         time_range: &TimeRange,
         pool: &T,
-    ) -> MetricsResult<Vec<(SdkEventMetricsBucketIdentifier, SdkEventMetricRow)>> {
+    ) -> MetricsResult<Vec<(AuthEventMetricsBucketIdentifier, AuthEventMetricRow)>> {
         let mut query_builder: QueryBuilder<T> = QueryBuilder::new(AnalyticsCollection::SdkEvents);
```

Comment:
we are querying the SdkEvents table but deserializing the results to `AuthEventMetricRow`...
I'd prefer deserializing to SdkEventMetricRow & then having an `Into::into` implementation to get `AuthEventMetricRow`...

Reply:
Since Authentication Analytics currently does not support filters/dimensions, the AuthEventMetricRow would only have time_bucket and count, so we can retain this